spec/system/consumer/split_checkout_spec.rb

```diff
@@ -223,6 +223,25 @@
       end
     end
 
+    context "with a saved address" do
+      let!(:address_state) do
+        create(:state, name: "Testville", abbr: "TST", country: DefaultCountry.country )
+      end
+      let(:saved_address) do
+        create(:bill_address, state: address_state, zipcode: "TST01" )
+      end
+
+      before do
+        user.update_columns bill_address_id: saved_address.id
+      end
+
+      xit "pre-fills address details" do
```

Comment:
I find the `pending` method useful in this case. Without `xit` the whole spec is still executed, then `pending` confirms that it really fails and you can add a comment to explain why you are doing this.